src/main/java/de/butbetter/keksys/CalculationController.java

```diff
@@ -17,6 +17,12 @@
     //@Autowired
     //private Calculator calculator;
 
+    /**
+     * GET-Request for VAT Calculation
+     * @param input = 0 - Float.Max
+     * @param percent = 0 - Float.Max
+     * @return ResponseEntity if all correct you will get HttpStatus.OK and VAT result as json
+     */
     @GetMapping("/VAT")
     public ResponseEntity<Float> calculation(@RequestParam(value = "input") Float input, @RequestParam(value = "percent") Float percent) {
```

Comment:
Why use Float , not Double?

Reply:
Addressed in here
https://github.com/But-Better/CalcApi/pull/6#discussion_r742679269

Reply:
Same as  Interface VAT